### DISCUSSION TITLE
Disable Chat preview

### DISCUSSION THREAD

**Discussion by joaosilva1002** [LiveHelperChat/livehelperchat#1944] [General]
Hello!

how can I disable this option so that operators can't see messages from chats that are still in the queue?

![image](https://user-images.githubusercontent.com/93102624/234577695-1e4cd7c9-b066-468a-b509-0ac01ef3fdac.png)

**Comment by remdex** [comment]
That alone option you can't disable but you can hide pending chats. 

https://doc.livehelperchat.com/docs/chat/list

**Comment by joaosilva1002** [comment]
Thank you!
